assets/src/edit-story/app/prepublish/guidance/media.js

```diff
@@ -144,12 +144,13 @@ export function videoElementLength(element) {
  * @param {Element} element The element being checked
  * @return {Guidance|undefined} The guidance object for consumption
  */
-export function videoElementOptimized(element) {
-  const idOrigin = element?.resource?.id?.split(':')?.[0];
+export function videoElementOptimized(element = {}) {
+  const idResource = element.resource?.id;
+  const idOrigin = idResource.toString().split(':')?.[0];
```

Comment:
The resource id is sometimes a number so I pulled this out into it's own variable and am turning it to a string just in case. It seems this happens when you upload via wordpress instead of in the editor.

Reply:
maybe add an extra check since idResource is accessed from optionally chaining:
```
const idOrigin = idResource?.toString().split(':')?.[0];
```